### PR TITLE
Feat: Allow Normalized Theta Tooltip

### PR DESCRIPTION
**Pull request by lsh**
Handles #8301, allows for the use of normalized percentage tooltips in `theta` encodings.

### CODE REVIEW

**Comment by kanitw** [comment]
Add `arc_pie_normalize_tooltip.vl.json` as an example?

- You can copy arc_pie.vl.json in `examples/specs`
- Under "Circular Plots" in examples.json, add 

```
{
        "name": "arc_pie_normalize_tooltip",
        "title": "Pie Chart with percentage_tooltip"
      },
```

File: src/channeldef.ts [hunk 1]
```diff
@@ -463,7 +463,7 @@ export interface PositionBaseMixins {
    *
    * `stack` can be one of the following values:
    * - `"zero"` or `true`: stacking with baseline offset at zero value of the scale (for creating typical stacked [bar](https://vega.github.io/vega-lite/docs/stack.html#bar) and [area](https://vega.github.io/vega-lite/docs/stack.html#area) chart).
-   * - `"normalize"` - stacking with normalized domain (for creating [normalized stacked bar and area charts](https://vega.github.io/vega-lite/docs/stack.html#normalized). <br/>
+   * - `"normalize"` - stacking with normalized domain (for creating [normalized stacked bar and area charts](https://vega.github.io/vega-lite/docs/stack.html#normalized)). <br/>
```
Comment (kanitw) [inline src/channeldef.ts:466]: Probably worth adding "`and pie charts [with percentage tooltip](https://vega.github.io/vega-lite/docs/arc.html#tooltip)`" in the parentheses.  

> - `"normalize"` - stacking with normalized domain (for creating [normalized stacked bar and area charts](https://vega.github.io/vega-lite/docs/stack.html#normalized))

File: site/docs/mark/arc.md [hunk 2]
```diff
@@ -67,6 +67,10 @@ You can also add a text layer to add labels to a pie chart.
 
 **Note:** For now, [you need to add `stack: true`](https://github.com/vega/vega-lite/issues/5078) to theta to force the text to apply the same polar stacking layout.
 
+Alternatively, setting `stack: "normalize"` allows for tooltips that display the percentage of the pie taken up by a each slice.
```
[the remark this answers is not in the document]
Comment (lsh) [reply site/docs/mark/arc.md:70]: Would this replace the note text on line 68 as well?

[the remark this answers is not in the document]
Comment (kanitw) [reply site/docs/mark/arc.md:70]: > Would this replace the note text on line 68

I don't think so. That's about labeling.  (That said, we can add 

`### Pie Charts with Labels` above that part too so the doc looks like having more structure

**Comment by lsh** [comment]
Closed in favor of #8308

**Pull request closed without merging**